packages/components/src/summary-button/docs/example.tsx

```diff
@@ -0,0 +1,20 @@
+import { Icon } from '@wordpress/components';
+import { backup } from '@wordpress/icons';
+import SummaryButton from '../index';
+
+function SummaryButtonExample() {
```

Comment:
@jameskoster due to the transparency and the background image, I noticed in the example (http://calypso.localhost:3000/devdocs/design/summary-button) that on hover we set the opacity too low (2%). 

Is that something we should change or it's fine since we don't expect such use cases.

Reply:
That link doesn't work for me so not 100% sure what you're referring to. The 2% opacity on hover is intentional to create sufficient contrast with the badges.

Reply:
You have to start the development server with `yarn start` first.

Reply:
Got it. I see the issue. It's *probably* okay as we don't expect `SummaryButton` to appear on dark backgrounds. But to be safe we could apply the same technique we used for the Badge component.

Reply:
I pushed an adjustment, let me know what you think.